sass/_functions.scss

```diff
@@ -0,0 +1,11 @@
+@function calc-rem($size) {
```

Comment:
Curious what the purpose of this function is. Why not just use `rem` units?

Reply:
I feel more natural to use `px`, matter of preference I suppose. The ideal though would be to use a limited set of size tokens (a "palette"). I _feel_ that it'll be easier to gather the set of sizes being used and decide on the "palette" if these are expressed on `px`.

Reply:
I tend to prefer px too, however, I also think it's generally a good idea to stick with the patterns that are established in a codebase unless you want to start a conversation about px vs rem.

Eventually I think it would be a good idea to create a theme file with arrays of preset values. So instead of using `16rem` you'd use variables like `font-sizes-0` and `font-sizes-1`. That way values stay standardized across the codebase, except for special cases.

Reply:
Fair enough. On the smaller PRs I'm using `rems` instead of `calc-rem`. I'll change this branch also to use `rem`. I'll leave this comment open as a reminder.

Reply:
I've removed the `calc-rem` usage from the components, I've left it on `html` though, I think that's acceptable.